### PR TITLE
DWPF-1799 Homepage Banner

### PR DESCRIPTION
**Pull request by 0RWilliams**
- [x] JIRA ticket referenced in title
- [x] Title is clear and concise
- [x] Description gives any relevant detail
- [x] Tests are up to date
- [x] Documentation is up to date

### CODE REVIEW

**Review by alsyx — APPROVED** [review]

File: src/dw_design_system/dwds/components/banner.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% extends "dwds/elements/extendable/card.html" %}
```
Comment (CamLamb) [inline src/dw_design_system/dwds/components/banner.html:1]: Can we change this file name to `banner_card.html`

File: src/dw_design_system/dwds/components/banner.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.dwds-banner-card {
```
Comment (CamLamb) [inline src/dw_design_system/dwds/components/banner.scss:1]: Can we change this file name to `banner_card.scss`

**Review by CamLamb — APPROVED** [review]
Just a couple of suggestions, otherwise LGTM!

File: src/dw_design_system/dwds/components/banner.html (hunk 1 above)
Comment (0RWilliams) [reply src/dw_design_system/dwds/components/banner.html:1]: Done.

File: src/dw_design_system/dwds/components/banner.scss (hunk 2 above)
Comment (0RWilliams) [reply src/dw_design_system/dwds/components/banner.scss:1]: Done.

**Pull request merged**